osv/models.py

```diff
@@ -666,6 +666,15 @@ def to_vulnerability(self, include_source=False):
       if credit.type:
         cr.type = vulnerability_pb2.Credit.Type.Value(credit.type)
       credits_.append(cr)
+    related_bugs = Bug.query(Bug.related == self.db_id).fetch()
+    if related_bugs:
+      related_bug_ids = [bug.db_id for bug in related_bugs]
+      self.related = sorted(list(set(related_bug_ids + self.related)))
+
+    alias_group = AliasGroup.query(AliasGroup.bug_ids == self.db_id).get()
+    if alias_group:
+      self.aliases = sorted(list(set(alias_group.bug_ids) - {self.db_id}))
+      modified = max(self.last_modified, alias_group.last_modified)
```

Comment:
nit: Make a new variable instead of modifying `self.related` and `self.aliases` here, we don't want to accidentally change what's stored in the raw aliases/related